### PR TITLE
Release notes for 1.6.2 draft

### PR DESCRIPTION
**Pull request by George-Aeillo**
Release Notes For 1.6.2

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -3,6 +3,15 @@ title: Release Notes
 weight: 2
 draft: false
 ---
+## 1.6.2
+### Features:
+- Analyze tool enhancements
+
+### Fixes:
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:10]: Please put a period at the end of each of the following bulleted items/sentences.

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -3,6 +3,15 @@ title: Release Notes
 weight: 2
 draft: false
 ---
+## 1.6.2
+### Features:
+- Analyze tool enhancements
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:8]: -> Verrazzano analysis tooling enhancements.

File: content/en/docs/releasenotes/_index.md [hunk 3]
```diff
@@ -3,6 +3,15 @@ title: Release Notes
...
+
+### Fixes:
+- Fixed issue with vz upgrade requiring --version when manifest was specified
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:11]: -> Fixed an issue with `vz upgrade` (code font (surrounded by back ticks)) requiring `--version` (code font) when a manifest file was specified.

File: content/en/docs/releasenotes/_index.md [hunk 4]
```diff
@@ -3,6 +3,15 @@ title: Release Notes
...
+- Fixed issue with vz install/upgrade processing CAPI image in air-gap environment
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:12]: -> Fixed an issue with `vz` (code font) install and upgrade processing CAPI images in a disconnected environment. (We don't/have not yet used the term air-gap.)

File: content/en/docs/releasenotes/_index.md [hunk 5]
```diff
@@ -3,6 +3,15 @@ title: Release Notes
...
+- Fixed issue with image format on pulls that prevented installation on AKS from working
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:13]: -> Fixed an issue with image format on pulls that prevented Verrazzano installation on AKS.

**Review by rosemarymarano — COMMENTED** [review]
Edits; please know me know they are fixed.

File: content/en/docs/releasenotes/_index.md (hunk 5 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:13]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 4 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:12]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 3 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 1 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:10]: Thanks!

**Pull request merged**